xcode/Subconscious/Subconscious.xcodeproj/project.pbxproj

```diff
@@ -1562,6 +1567,28 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
+/* Begin PBXShellScriptBuildPhase section */
+		8804D2562A424FD900D45E83 /* Run Script */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+				"${DWARF_DSYM_FOLDER_PATH}/${DWARF_DSYM_FILE_NAME}/Contents/Resources/DWARF/${TARGET_NAME}",
+			);
+			name = "Run Script";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "if which sentry-cli >/dev/null; then\nexport SENTRY_ORG=subconscious\nexport SENTRY_PROJECT=apple-ios\nexport SENTRY_AUTH_TOKEN=cfb08a61febc418bb55ec80ae6f99e2ce34b93d5b179454e9a8b8c240a3bb4d6\nERROR=$(sentry-cli debug-files upload \"$DWARF_DSYM_FOLDER_PATH\" 2>&1 >/dev/null)\nif [ ! $? -eq 0 ]; then\necho \"warning: sentry-cli - $ERROR\"\nfi\nelse\necho \"warning: sentry-cli not installed. download from https://github.com/getsentry/sentry-cli/releases\"\nfi\n";
```

Comment:
Is it okay for this auth token to be publicly checked in? Bearing in mind this repo is fully open-source.

This issue seems to suggest we shouldn't check it in: https://github.com/getsentry/sentry-react-native/issues/1096